facets.wdl

```diff
@@ -16,15 +16,19 @@ workflow facets_workflow {
 
     Int? cval = 150
     Int? maxiter = 10
+    Int? seed_initial = 42
+    Int? seed_iterations = 10
 
     Int? pileup_memoryGB = 3
     Int? pileup_diskGB = 200
```

Comment:
Can this be dynamically sizes?
`pileup_diskGB = ceil(1.1 * (size(normal_bam + 'G') + size(tumor_bam, 'G'))) + 2`